src/components/CardRestSlider.tsx

```diff
@@ -37,10 +37,11 @@ const CardRestSlider: React.FC = () => {
       {status === "succeeded" && (
         <Slider {...SLIDER_SETTINGS}>
           {restaurants.map((card: ICard) => (
-            <Card key={card._id} card={card} />
+            <Link to={`/restaurants/${card._id}`} key={card._id}>
+              <Card key={card._id} card={card} />
+            </Link>
```

Comment:
In my opinion, it would be better to wrap the Card component with a Link element from inside the component itself as anyway you pass the card object to it